### DISCUSSION TITLE
Issue with importing ecoinvent (+solution)

### DISCUSSION THREAD

**Discussion by mkvdhulst** [LCA-ActivityBrowser/activity-browser#781] [Q&A]
I have been trying to import ecoinvent using the Import remote data (download) option in the Database Import Wizard. After successfully logging in, I tried to proceed with downloading but got the following message: "Cannot find any valid data with the given login credentials". 

After some trial and error, I found out that I had to login to the ecoinvent database on their website and accept the latest EULA. This solved the issue for me and I was able to import ecoinvent using the Database Import wizard. Thought I'd leave my findings here in case others get stuck on the same issue.

**Comment by marc-vdm** [comment]
ecoinvent has a completely new system for AB to interact with their website, so I'm closing this as it's no longer relevant.
